public/components/FormControls/FormikInputWrapper/FormikInputWrapper.js

```diff
@@ -15,7 +15,7 @@ const FormikInputWrapper = ({ name, fieldProps, render }) => (
 
 FormikInputWrapper.propTypes = {
   name: PropTypes.string.isRequired,
-  fieldProps: PropTypes.object.isRequired,
+  fieldProps: PropTypes.object,
```

Comment:
Why do we not want to require `fieldProps` anymore?

Reply:
We are not passing this prop from all parents and that throws a big warning in the console